superset-frontend/plugins/plugin-chart-echarts/src/Funnel/transformProps.ts

```diff
@@ -227,13 +229,13 @@ export default function transformProps(
     },
     tooltip: {
       ...getDefaultTooltip(refs),
-      show: !inContextMenu,
+      show: showTooltipLabels,
```

Comment:
`inContextMenu` is used to avoid showing the tooltip when right clicking.
```suggestion
      show: !inContextMenu & showTooltipLabels,
```